src/Components/Home.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaGlobe } from 'react-icons/fa';
+import { fetchCountries } from '../redux/slices/countriesSlice';
+
+function HomePage() {
+  const { countries } = useSelector((state) => state.countries);
+  const [searchTerm, setSearchTerm] = useState('');
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, [dispatch]);
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  let filteredCountries = countries;
+  if (searchTerm) {
+    filteredCountries = countries.filter(
+      (country) => country.name.toLowerCase().includes(searchTerm.toLowerCase()),
+    );
+  }
+
+  return (
+    <section className="home-page">
+      <div className="flex items-center justify-between p-4">
+        <h1 className="text-2xl font-bold text-gray-800">
+          <FaGlobe className="inline-block mr-2" />
+          Countries
+        </h1>
```

Comment:
- [ ] Your design doesn't have the top section compared to that of the template provided for the home page. It would be nice to have a top section based on the design provided in the Behance template. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/220474479-82bb1fd7-c216-48a5-ae15-b1e170f7b1ae.png)


### Behance Template

![image](https://user-images.githubusercontent.com/77106962/220474553-80403ac9-dbba-4b7b-b3bc-549a989ec203.png)